### ISSUE TITLE
Custom loss function

### ISSUE THREAD

**Issue by yug125lk** [aimansnigdha/Ambiguous-Medical-Image-Segmentation-using-Diffusion-Models#6]
Hi, thank you for sharing this impressive work! I have a question: how can I train the model using a custom loss function that compares the generated image with the ground truth? Thank you in advance.
